packages/blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Account/index.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { MouseEvent, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { IconCheckCircle, IconCopy } from '@blockchain-com/icons'
+import Tooltip from 'blockchain-wallet-v4-frontend/src/scenes/plugin/SwitchAccount/Tooltip'
+import styled from 'styled-components'
+
+import { Icon, Text } from 'blockchain-info-components'
+import { getBalanceSelector } from 'components/Balances/selectors'
+import FiatDisplay from 'components/Display/FiatDisplay'
+import { RootState } from 'data/rootReducer'
+
+import { SwapAccountType } from '..'
+
+const AccountBlock = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 22px 0;
+  cursor: pointer;
+`
+const ConnectBlock = styled.div`
+  position: relative;
+  #tooltip {
+    display: none;
+  }
+  &:hover {
+    #tooltip {
+      display: block;
+    }
+  }
+`
+const AccountInfo = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  margin: auto auto auto 16px;
+  #tooltip {
+    display: none;
+  }
+  &:hover {
+    #tooltip {
+      display: block;
+    }
+  }
+`
+const IconCheckWrapper = styled(IconCheckCircle)`
+  path {
+    fill: ${(props) => props.theme.white};
+  }
+`
+const IconUncheckCircle = styled.div`
+  width: 24px;
+  height: 24px;
+  border-radius: 50%;
+  border: 2px solid ${(props) => props.theme.grey800};
+`
+const WalletBlock = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  cursor: copy;
+`
+class TooltipProperties {
```

Comment:
Are you adding a new generic component?

I'd rather not add `classes`, as nowhere else in the app do we use this standard. Just use a function instead?